src/foundation/layout.css

```diff
@@ -1,5 +1,15 @@
 :root {
-  --page-padding-x: 6rem;
+  --main-content-max-width: 100rem;
```

Comment:
It's 1600px, it's too much. After 800px text becomes hard to read because it's too wide.

Reply:
What do you suggest then? Something in between like 1200px? This value is quite important because it cannot be changed in the application. It has to be changed here and it applies to all subprojects.

Reply:
1200 is fine.